lib/sedna/algorithms/multi_task_learning/task_jobs/task_relation_discover.py

```diff
@@ -12,16 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import tensorflow as tf
+from typing import List
+from .artifact import Task, TaskGroup
+from sedna.common.class_factory import ClassType, ClassFactory
 
-from sedna.common.config import BaseConfig
 
+__all__ = ('DefaultTaskRelationDiscover', )
 
-def load_model():
-    model_url = BaseConfig.model_url
-    return tf.keras.models.load_model(model_url)
 
+@ClassFactory.register(ClassType.MTL)
+class DefaultTaskRelationDiscover:
+    def __init__(self, **kwargs):
+        pass
 
-def save_model(model):
-    model_url = BaseConfig.model_url
-    model.save(model_url)
+    def __call__(self, tasks: List[Task]) -> List[TaskGroup]:
+        tg = []
```

Comment:
tg is list of TaskGroup, better to use tgs

Reply:
fix